frontend/src/component/admin/groups/GroupForm/GroupFormUsersSelect/GroupFormUsersSelect.tsx

```diff
@@ -44,7 +46,11 @@ const renderOption = (
         />
         <StyledOption>
             <span>{option.name || option.username}</span>
-            <span>{option.email}</span>
+            <span>
+                {option.name && option.username
+                    ? option.username
+                    : option.email}
```

Comment:
I understand the logic here is: 
- If we've shown the name in the previous span and we also have a username, just display the username. 
- Otherwise, just show the email

This leaves us with either:
1. name + username (if both are present)
2. name + email (if username is not present)
3. username + email (if name is not present)

I think it'd be simpler if we just have conditional spans for each one of the values. Would that be an option?

Reply:
Can you show an example of what you have in mind?

It's important to keep the total spans as 2, a main "title" one and a "subtitle" one, like the screenshot I shared above in the PR description. We need to cover the following scenarios:

1. Service accounts have a name and username;
2. Normal user accounts usually have a name and email;
3. Accounts like `admin` don't have a name or email, only username;

Reply:
Got it, so what I had in mind is potentially going to 3 spans, but if that's a restriction then forget about it. The only I'd consider is https://github.com/Unleash/unleash/pull/2917#discussion_r1073296806